### DISCUSSION TITLE
Recurrent cleanup notifications

### DISCUSSION THREAD

**Discussion by geekmiki** [Cookie-AutoDelete/Cookie-AutoDelete#1083] [Q&A / Support] [answered]
Hi,

I am using CAD 3.6.0 on FF 86.0.1.

I have a recurrent clean notification showing up every time I close a tab:
Some site data, including x Cookie(s), removed from x domain(s), including: domain1, domain2, etc
domain1: Clean because domain1 is not in the White or Grey lists

These domains are not open in any tab and have never been white- oy greylisted. I have visited them at some point, but the cookies should have been deleted a long time ago. 
If I visit the website in question again, it's as if that particular cookie gets "stuck" and doesn't get deleted. So the notification will show up continuously. If I delete the cookie manually, it'll be ok until I visit that site again. This happens on random sites and those cookies get stuck. Last time I had over 100 and had to delete them manually.

Any clue on why this happens?

**Comment by kennethtran93** [comment, marked as answer]
Possible duplicate of #807. Have a read there and see if it applies to you as well.
